Signal/src/ViewControllers/MediaMessageView.swift

```diff
@@ -18,8 +18,15 @@ class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
     var audioPlayer: OWSAudioAttachmentPlayer?
     var audioStatusLabel: UILabel?
     var audioPlayButton: UIButton?
-    var isAudioPlayingFlag = false
-    var isAudioPaused = false
+    var playbackState = AudioPlaybackState.stopped {
```

Comment:
I've streamlined the `OWSAudioAttachmentPlayerDelegate` interface, which makes our audio player views simpler.